packages/suite/src/components/suite/Settings/components/TextColumn.tsx

```diff
@@ -27,14 +27,16 @@ const Description = styled.div`
     margin-bottom: 12px;
     margin-top: 12px;
     font-size: ${variables.FONT_SIZE.TINY};
```

Comment:
```suggestion
    font-size: ${variables.FONT_SIZE.SMALL};
```

According to Zeplin, this should be 14px. You can modify Cypress tests, if it is required 😉

Reply:
It is required to modify test as bigger font also changes the height of the application log modal